spec/rspec/core/formatters/base_text_formatter_spec.rb

```diff
@@ -30,6 +35,15 @@
       formatter.close(RSpec::Core::Notifications::NullNotification)
       expect(output_to_close.closed?).to be(false)
     end
+
+    it "restores the output's sync setting" do
+      if !output_to_close.respond_to?(:sync=)
+        skip "output_to_close does not support sync, so closing the formatter will not reset it"
```

Comment:
In what scenario is this the case? Generally we don't define test cases like this that can conditional fail unless its ruby version related, and then we gate the spec on the existance of the feature so this should at least explain that,

Reply:
It was a guard against a version of ruby that I couldn't test with not supporting `sync` on File. But I'll remove it and just re-add if that is the case after a CI run.